tests/conftest.py

```diff
@@ -21,6 +21,10 @@
         os.environ.get('GH_AUTH', 'x' * 20)
         )
 
+    config.default_cassette_options['match_requests_on'].extend([
+        'json-body'
```

Comment:
> Could also just be .append('json-body').

Gotcha. Referred to this document (http://betamax.readthedocs.org/en/latest/matchers.html#specifying-matchers) and wasn't sure if that was the preferred method